webapp/src/pages/RegisterPage/RegisterPage.tsx

```diff
@@ -91,31 +74,28 @@ export default function RegisterPage() {
       await authApi.authControllerRegisterLocalUser({
         registerUserDto: formValues,
       });
-      setStatus({
-        type: 'success',
-        message: 'Registration complete',
-      });
+
+      notify('Registration complete');
+
       setTimeout(() => {
         navigate(LOGIN_OPTIONS_URL);
       }, 2000);
```

Comment:
```suggestion
      navigate(LOGIN_OPTIONS_URL);
```

suggestion (if-minor): We could remove the timeout because this notification persists even if the page changes :heart:

Reply:
Nice idea!! done